user-guide/Reference/Soft-launch_options/Overview_of_Soft_Launch_Options.md

```diff
@@ -216,6 +216,9 @@ Enables the use of Elasticsearch for alarm indexing and other applications. To p
 - **Minimum version**: 9.5.0
 - **Release version**: 10.0.0/1.0.0.2 ([RN 24054](xref:General_Feature_Release_10.0.2_new_features#dataminer-indexing-id_13370id_13406id_13504id_13571id_13623-id_13622id_13629id_13695id_13769id_13912id_14001id_14038-id_16287id_16896id_16915id_16935id_16959id_17081id_17166-id_17328id_17851id_18562id_18714id_19337id_19437id_19443-id_19691id_20373id_20845id_20998id_21205id_21257id_21634-id_22378id_22927id_23049id_23998id_24054id_24158))
 
+> [!NOTE]
+> From DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/10.4.6 onwards<!--RN 39173-->, the *Indexing* application is no longer available when you are using a [DaaS system](xref:Creating_a_DMS_in_the_cloud), as database configuration is included as part of DaaS.
```

Comment:
I don't think it's possible to create a DaaS system with such an old DataMiner version, so this note is probably not necessary.

Reply:
The soft launch is still active in Cube, so the possibility that  a user can use it on DaaS, is nihil yet relevant.

Reply:
Some testing (thanks, @EdithLansens 😊) showed that the argument indeed still shows the Indexing app in recent systems, even though this soft-launch option is documented as having been released – so apparently this particular aspect of it was never released. Do you know if there are any plans to ever release it? Because if this option has been abandoned, it might be better if we added that info instead, so people do not get encouraged to still use this.

Reply:
RN [24054] refers to a feature within the Alarm Console, not to the module.
I don't believe this will ever be released.. I did take it into account though, just for certainty.